turbustat/statistics/pca/pca.py

```diff
@@ -209,3 +834,72 @@ def distance_metric(self, verbose=False, label1="Cube 1", label2="Cube 2"):
             p.show()
 
         return self
+
+
+@np.vectorize
```

Comment:
Given the simplicity of the equations below, I think `vectorize` is unnecessary and might even slow down the code.